Frameworks/include/CGGradientInternal.h

```diff
@@ -34,3 +34,11 @@ class __CGGradient : private objc_object {
     void initWithColorComponents(const float* components, const float* locations, size_t count, CGColorSpaceRef colorspace);
     void initWithColors(CFArrayRef components, const float* locations, CGColorSpaceRef colorspace);
 };
+
+unsigned long _CGGradientGetCount(CGGradientRef gradient);
+
+float* _CGGradientGetStopLocation(CGGradientRef gradient);
+
+float* _CGGradientGetColorComponents(CGGradientRef gradient);
```

Comment:
Agreed, but again it's internal access + we can wrap it at a cost,

---
In reply to: [92450607](https://github.com/Microsoft/WinObjC/pull/1535#discussion_r92450607) [](ancestors = 92450607)